doc/man3/SSL_get_value_uint.pod

```diff
@@ -1,353 +0,0 @@
-=pod
-
-=head1 NAME
-
-SSL_get_value_uint, SSL_set_value_uint, SSL_get_generic_value_uint,
-SSL_set_generic_value_uint, SSL_get_feature_request_uint,
-SSL_set_feature_request_uint, SSL_get_feature_peer_request_uint,
-SSL_get_feature_negotiated_uint, SSL_get_quic_stream_bidi_local_avail,
```

Comment:
I have no idea what this is supposed to be good for but you can never add too many knobs...

In any case, the first few of these seem generic and not necessarily quic related, so I'm unsure if these should be deleted.

Reply:
The only thing they were ever used for is to query parameters on QUIC objects so I can't see a reason to keep them.